src/main/host/thread_protected.h

```diff
@@ -52,6 +52,8 @@ struct _Thread {
     // Non-null if blocked by a syscall.
     SysCallCondition* cond;
 
+    int affinity;
```

Comment:
Nit: comment to explain what this is storing?

Reply:
fixed